subworkflows/local/vcf_roi_tagging/main.nf

```diff
@@ -0,0 +1,91 @@
+
+include { PREPAREBEDFILE                            } from '../../../modules/local/bedfile/preparebedfile/main'
+include { TABIX_BGZIP as FIRST_COMPRESS             } from '../../../modules/nf-core/tabix/bgzip/main'
+include { TABIX_BGZIP as SECOND_COMPRESS            } from '../../../modules/nf-core/tabix/bgzip/main'
+include { TABIX_TABIX                               } from '../../../modules/nf-core/tabix/tabix/main'
+include { BCFTOOLS_INDEX                            } from '../../../modules/nf-core/bcftools/index/main'
+include { BCFTOOLS_INDEX as TAGROI_INDEX            } from '../../../modules/nf-core/bcftools/index/main'
+include { BCFTOOLS_ANNOTATE as BCFTOOLS_ADDROI      } from '../../../modules/nf-core/bcftools/annotate/main'
+include { BCFTOOLS_ANNOTATE as BCFTOOLS_RMVROI      } from '../../../modules/nf-core/bcftools/annotate/main'
+include { VEMBRANE_TAG as FILTER_TAGROI             } from '../../../modules/local/vembrane/tag/main'
+
+workflow TAGROI {
+
+    take:
+    bedfile             // path to BED file
+    vcf_tbi             // channel: [ val(meta), path(vcf), path(tbi) ]
+
+    main:
+
+    ch_versions = Channel.empty()
+    annotation_header = Channel.fromPath("$projectDir/bin/header_line.txt").collect()
```

Comment:
i dont like that we carry a whole file here for a one-line string.
can we simply hardcode the string here instead?

Reply:
I've tried to do so, but it didn't work. I've read the documentation for [bcftools annotate](https://samtools.github.io/bcftools/bcftools.html#annotate) and it expects a file containing the header information. I guess we have to carry it along - if you've got an idea on how to circumvent this, I'd be happy to implement it!

Reply:
If its not possible to hardcode with groovy (probably because the file location cannot be properly specified) I have an (not ideal) alternative:
Create a process to create this one-line string. So the process would simply `echo` the one-liner into a file and output it as channel.

Reply:
So we would need to choose between an additional module creating a temporary file or a single-line txt file just for this step? 
I would stick to the latter and would propose to push this point down the line as a "beautification/enhancement" issue as the current implementation is working as intended.